resolvers/context.js

```diff
@@ -0,0 +1,29 @@
+const cookie = require('cookie')
+const jwt = require('jsonwebtoken')
+const { User } = require('../models/User')
+const { findOrCreate } = require('../models/utils')
+
+const setContext = async ({ req }) => {
+  let decoded
+
+  try {
+    const authCookieName = process.env.SSO_COOKIE_NAME
```

Comment:
We should find a way to mock these things and not rely on users environment variables for testing

Reply:
I have address it at 00bd066